Makefile

```
@@ -21,7 +21,7 @@ dependency: ## Get dependencies
 lint: dependency ## Lint the files
 	@echo "Running linter..."
 	@if [ ! -f "$(GOPATH)/bin/golangci-lint" ] && [ ! -f "$(shell go env GOROOT)/bin/golangci-lint" ]; then \
-		$(GO) install github.com/golangci/golangci-lint/cmd/golangci-lint@v1.59.1; \
+		$(GO) install github.com/golangci/golangci-lint/cmd/golangci-lint@latest; \
```

```suggestion
		$(GO) install github.com/golangci/golangci-lint/cmd/golangci-lint@1.60.2; \
```

`golangci-lint`의 버전을 명시하여 관리하는 편이 좋겠다는 의견을 말씀드립니다.

`golangci-lint` Changelog 로 볼때, v1.60.2를 적용하면, Go 1.23.0에 대한 lint를 지원할 수 있겠네요. 

이후 커뮤니티 차원의 Go 버전을 업그레이드시에 알맞은 `golangci-lint` 버전을 함께 인하여 업그레이드 해 나가면 될 것 같고요. 

개인적으로는 `golangci-lint`에서 지원하지 않는 Go 버전으로의 업그레이드는 재고해볼 필요가 있겠다는 생각도 들었습니다.

(참고) Changelog - https://golangci-lint.run/product/changelog/

> Changelog
> Follow the news and releases on [Mastodon](https://fosstodon.org/@golangcilint) and on [Twitter](https://twitter.com/golangci).

> 
> v1.60.2
> (중략)
> misc.
> 🎉 go1.23 support